### DISCUSSION TITLE
UVR5 process model list?

### DISCUSSION THREAD

**Discussion by DpaulKu** [Anjok07/ultimatevocalremovergui#1222] [Q&A]
Hi, I'm just getting started with UVR5 and am wondering if it's OK to ask... is there a list out there of the available processing models and a brief explanation of their intended use? I'm not a good waiter and it looks to me to be a "one at a time" proposition for scores of cryptically named models. Sorry if this has been asked and answered. A pointer to there would be fine. Thanks

**Comment by ZaCloud** [comment]
Seconding this. It's kind of frustrating that we can only download one method/model at a time, & most of them have weird code-looking names (or even human names, LOL) that don't do a good job telling what they actually do. A more user-friendly presentation would help a lot.

But of course in the meantime, just a list with short summaries would be appreciated. ^^;;

Unguided, this is a daunting amount of trial-&-error, considering one may have to also try multiple setting variations on each of those as well, to truly know if each method & model can or can't do the task we want. :(

**Comment by DpaulKu** [comment]
OK, I get it. Within the GitHub sphere there may me little tolerance for nOObies. I did find this general tutorial: https://multimedia.easeus.com/ai-article/how-to-use-ultimate-vocal-remover.html  

Also a Reddit thread here:  https://www.reddit.com/r/IsolatedTracks/comments/vuavwq/ultimate_vocal_remover/  

'Back to the shadows...
